plugin.video.itvrs/addon.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.itvrs"
+	name="ITVRS"
+	version="1.0.0"
+	provider-name="ITVRS">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.addon.common" version="2.0.0"/>
+    <import addon="plugin.video.youtube" version="5.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">ITVRS TV And Radio video Add-on</summary>
```

Comment:
since the addon provides content, please fill the `language` tag with the respective language of the content. In this case I suppose it should be `en` only.